### ISSUE TITLE
Voice dictation shortcut

### ISSUE THREAD

**Issue by BeReasonable** [microsoft/vscode#206800]
Type: <b>Bug</b>

Thanks for adding dictation! Features like this are great for users with limited mobility (or those looking for a performance boost)

Ask: can you update the shortcut so it is consistent with W/X/P?
In those apps it is Alt+`

A great accessibility experience includes having consistency across workflows. Thanks for investing in features like these

VS Code version: Code 1.87.0 (019f4d1419fbc8219a181fab7892ebccf7ee29a2, 2024-02-27T23:41:44.469Z)
OS version: Windows_NT x64 10.0.22631
Modes:

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|AMD Ryzen 9 5900X 12-Core Processor             (24 x 3700)|
|GPU Status|2d_canvas: enabled<br>canvas_oop_rasterization: enabled_on<br>direct_rendering_display_compositor: disabled_off_ok<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>opengl: enabled_on<br>rasterization: enabled<br>raw_draw: disabled_off_ok<br>skia_graphite: disabled_off<br>video_decode: enabled<br>video_encode: enabled<br>vulkan: disabled_off<br>webgl: enabled<br>webgl2: enabled<br>webgpu: enabled|
|Load (avg)|undefined|
|Memory (System)|31.92GB (15.85GB free)|
|Process Argv|--crash-reporter-id 14dd867b-75ec-4cb0-9d8b-b25e728286ca|
|Screen Reader|no|
|VM|0%|
</details><details><summary>Extensions (20)</summary>

Extension|Author (truncated)|Version
---|---|---
vscode-markdownlint|Dav|0.54.0
docs-article-templates|doc|1.0.3
docs-authoring-pack|doc|1.0.1
docs-build|doc|0.4.2
docs-markdown|doc|1.0.3
docs-preview|doc|1.0.3
docs-yaml|doc|1.0.1
kusto|don|0.4.4
vsc-material-theme|Equ|34.3.1
vsc-material-theme-icons|equ|3.5.0
prettier-vscode|esb|10.1.0
copilot|Git|1.171.0
copilot-chat|Git|0.13.0
Learn-Training-AI-Assistant|Lea|1.0.35
vscode-dotnet-runtime|ms-|2.0.2
vscode-edge-devtools|ms-|2.1.5
debugpy|ms-|2024.2.0
python|ms-|2024.2.1
vscode-yaml|red|1.14.0
kuskus-kusto-syntax-highlighting|ros|1.2.1

(1 theme extensions excluded)

</details><details>
<summary>A/B Experiments</summary>

```
vsliv368cf:30146710
vspor879:30202332
vspor708:30202333
vspor363:30204092
vscoreces:30445986
vscod805:30301674
binariesv615:30325510
vsaa593:30376534
py29gd2263:30899288
vscaat:30438848
c4g48928:30535728
azure-dev_surveyone:30548225
a9j8j154:30646983
962ge761:30959799
pythongtdpath:30769146
welcomedialogc:30910334
pythonidxpt:30866567
pythonnoceb:30805159
asynctok:30898717
pythontestfixt:30902429
pythonregdiag2:30936856
pyreplss1:30897532
pythonmypyd1:30879173
pythoncet0:30885854
pythontbext0:30879054
accentitlementst:30887150
dsvsc016:30899300
dsvsc017:30899301
dsvsc018:30899302
cppperfcontrol:30973184
d34g3935:30971562
edj9j583:30969295
a89i1917:30973842
bg6jg535:30958357
ccp2r6:30958158
dsvsc020:30973508

```

</details>

**Comment by meganrogge** [comment]
@bpasero , what do you think of this? 

When I reassigned the keybinding to be `alt+``, it did not work with this log:

```
 Ignoring single modifier alt due to it being pressed together with other keys.
```

**Comment by bpasero** [comment]
Yes that makes good sense to me, I was not aware of a standard keyboard shortcut.

@ulugbekna could you possibly assist @meganrogge with the keybinding trouble? Thanks1

**Comment by BeReasonable** [comment]
Correct. This is about the dictate command and not the copilot chat one. ctrl+alt+v

**Comment by ulugbekna** [comment]
@meganrogge 

> When I reassigned the keybinding to be alt+`, it did not work with this log:

Could you show all logs produced by keyboard troubleshooting when you press alt+` ? 

Could you also press alt+` at https://w3c.github.io/uievents/tools/key-event-viewer.html and show the output? I wonder if alt+` is a system shortcut that's not captured by OS and not passed to apps

**Comment by meganrogge** [comment]
```
2024-03-07 09:41:39.342 [info] [KeybindingService]: / Soft dispatching keyboard event
2024-03-07 09:41:39.343 [info] [KeybindingService]: | Resolving alt+[Backquote]
2024-03-07 09:41:39.344 [info] [KeybindingService]: \ From 1 keybinding entries, matched workbench.action.editorDictation.start, when: hasSpeechProvider && !editorDictation.inProgress && !editorReadonly, source: user.
2024-03-07 09:41:39.352 [info] [KeybindingService]: + Storing single modifier for possible chord alt.
2024-03-07 09:41:39.352 [info] [KeybindingService]: + Clearing single modifier due to other key up.
2024-03-07 09:41:43.636 [info] [KeybindingService]: / Soft dispatching keyboard event
2024-03-07 09:41:43.636 [info] [KeybindingService]: \ Keyboard event cannot be dispatched
2024-03-07 09:41:43.637 [info] [KeybindingService]: / Received  keydown event - modifiers: [meta], code: MetaLeft, keyCode: 91, key: Meta
2024-03-07 09:41:43.637 [info] [KeybindingService]: | Converted keydown event - modifiers: [meta], code: MetaLeft, keyCode: 57 ('Meta')
```

**Comment by meganrogge** [comment]
![Image](https://github.com/microsoft/vscode/assets/29464607/8480eb5a-b4e3-400d-a53f-4724ef17765c)

**Comment by meganrogge** [comment]
It doesn't end up working and the ` is sent to the editor. 

https://github.com/microsoft/vscode/assets/29464607/ad62ff01-4c24-4c49-a13f-57385d9d0a38

**Comment by BeReasonable** [comment]
Neither shortcut works. Ctrl-Alt-V doesn't trigger dictation. Neither does Alt-` when I remap it

**Comment by bpasero** [comment]
@BeReasonable you need to install https://marketplace.visualstudio.com/items?itemName=ms-vscode.vscode-speech

**Comment by BeReasonable** [comment]
Nice!

I've installed the extension, and Alt-` does work when I override the setting in vscode.

**Comment by bpasero** [comment]
@BeReasonable when you type that into the keybindings editor, how does it appear, literally as Alt+` ?

![image](https://github.com/microsoft/vscode/assets/900690/03555da0-9a96-4b7f-aa08-935aa4efe34f)

I am not on a US keyboard, so thats why I wonder.

**Comment by BeReasonable** [comment]
Yes

**Comment by bpasero** [comment]
@BeReasonable can you clarify where the keybinding is documented for Windows? I only found https://support.microsoft.com/en-us/windows/use-voice-typing-to-talk-instead-of-type-on-your-pc-fec94565-c4bd-329d-e59a-af033fa5689f#WindowsVersion=Windows_11 which talks about another keybinding?

**Comment by BeReasonable** [comment]
The Home ribbon in W/X/P/Outlook
[A screenshot of a computer  Description automatically generated]
